### DISCUSSION TITLE
Netflix双语字幕

### DISCUSSION THREAD

**Discussion by baihao2226** [DualSubs/.github#3] [Q&A 问答] [answered]
❗️❗️❗️一键安装后还需要怎么弄 Netflix上没有双语选项 到底怎么弄啊

**Comment by VirgilClyne** [comment, marked as answer]
https://dualsubs.github.io/guide/netflix

**Reply by baihao2226** [reply]
![image](https://github.com/user-attachments/assets/c2ddce31-c245-4cf3-98cf-44a6b7f9969a)
